### ISSUE TITLE
Problem: custom emoji aren't rendered

### ISSUE THREAD

**Issue by dawran6** [clojureverse/clojurians-log-app#75]
An example is the `:sheepy:` custom emoji aren't rendered [here](https://clojurians-log.clojureverse.org/announcements/2019-09-20/1568966716.046800), while slack rendered it as:

```html
<span class="c-emoji c-emoji__medium c-emoji--inline" data-qa="emoji" delay="300" aria-describedby="slack-kit-tooltip">
  <img src="https://emoji.slack-edge.com/T03RZGPFR/sheepy/5de1946cce6cc187.gif" aria-label="sheepy emoji" alt=":sheepy:" data-stringify-type="emoji" data-stringify-emoji=":sheepy:">
</span>
```

We might be able to use the Slack API to list all the custom emoji: https://api.slack.com/methods/emoji.list

(This is related to #71.)

**Comment by dawran6** [comment]
Hi @plexus, I'd like to include some custom emoji data into the `clojurians-log-demo-data` so I can work on this issue without the slack API token. Would you help me to get the current list of emojis? This can be done with the `clj-slack.emoji` ns:

```clojure
(require '[clj-slack.emoji :as slack-emoji])
(:emoji (slack-emoji/list (slack-conn)))
```
